src/um/winbase.rs

```diff
@@ -1100,6 +1100,214 @@ extern "system" {
         lpdwSize: PDWORD,
     ) -> BOOL;
 }
+pub const PROC_THREAD_ATTRIBUTE_NUMBER: DWORD = 0x0000FFFF;
+pub const PROC_THREAD_ATTRIBUTE_THREAD: DWORD = 0x00010000;
+pub const PROC_THREAD_ATTRIBUTE_INPUT: DWORD = 0x00020000;
+pub const PROC_THREAD_ATTRIBUTE_ADDITIVE: DWORD = 0x00040000;
+pub enum PROC_THREAD_ATTRIBUTE_NUM {
+    ProcThreadAttributeParentProcess = 0,
+    ProcThreadAttributeHandleList = 2,
+    ProcThreadAttributeGroupAffinity = 3,
+    ProcThreadAttributePreferredNode = 4,
+    ProcThreadAttributeIdealProcessor = 5,
+    ProcThreadAttributeUmsThread = 6,
+    ProcThreadAttributeMitigationPolicy = 7,
+    ProcThreadAttributeSecurityCapabilities = 9,
+    ProcThreadAttributeProtectionLevel = 11,
+    ProcThreadAttributeJobList = 13,
+    ProcThreadAttributeChildProcessPolicy = 14,
+    ProcThreadAttributeAllApplicationPackagesPolicy = 15,
+    ProcThreadAttributeWin32kFilter = 16,
+    ProcThreadAttributeSafeOpenPromptOriginClaim = 17,
+    ProcThreadAttributeDesktopAppPolicy = 18,
+}
+macro_rules! ProcThreadAttributeValue {
+    ($Number:path, $Thread:ident, $Input:ident, $Additive:ident) => {
+        ((($Number as DWORD) & PROC_THREAD_ATTRIBUTE_NUMBER) |
+            ((($Thread & 1) as DWORD) * PROC_THREAD_ATTRIBUTE_THREAD) |
+            ((($Input & 1) as DWORD) * PROC_THREAD_ATTRIBUTE_INPUT) |
+            ((($Additive & 1) as DWORD) * PROC_THREAD_ATTRIBUTE_ADDITIVE)) as DWORD_PTR
+    }
+}
+pub const PROC_THREAD_ATTRIBUTE_PARENT_PROCESS: DWORD_PTR = ProcThreadAttributeValue!(
+    PROC_THREAD_ATTRIBUTE_NUM::ProcThreadAttributeParentProcess,
```

Comment:
Which would in turn shorten this, allowing the entire `ProcThreadAttributeValue` invocation to be on a single new line, similar to the headers.